chat-plugins/uno.js

```diff
@@ -584,15 +722,16 @@ exports.commands = {
 
 		timer: function (target, room, user) {
 			if (!this.can('minigame', null, room)) return;
-			if (!room.game || room.game.gameid !== 'uno') return this.errorReply("There is no UNO game going on in this room.");
+			const game = /** @type {UnoGame} */ (room.game);
+			if (!game || game.gameid !== 'uno') return this.errorReply("There is no UNO game going on in this room.");
 			let amount = parseInt(target);
 			if (!amount || amount < 5 || amount > 300) return this.errorReply("The amount must be a number between 5 and 300.");
 
-			room.game.maxTime = amount;
-			if (room.game.timer) {
-				clearTimeout(room.game.timer);
-				room.game.timer = setTimeout(() => {
-					room.game.eliminate(room.game.currentPlayer);
+			game.maxTime = amount;
+			if (game.timer) {
```

Comment:
Not exactly sure whats happening here as I dont see any code thats setting the timer if it was not set previously, maybe this is one of the things typescript caught (timer was set to a non timer truthy value maybe).